### PR TITLE
add on/observes

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: addon/index.js [hunk 1]
```diff
@@ -71,3 +71,20 @@ export function readOnly(target, name, descriptor) {
...
+    } else {
+      return function(target, key, descriptor) {
+        descriptor.value = fn.apply(null, params.concat(descriptor.value));
+        return descriptor;
+      };
+    }
+  };
+}
+
+export var on = decoratorAlias(Ember.on, 'Connot `on` without event names');
```
Comment (rwjblue) [inline addon/index.js]: s/Connot/Can not/

File: addon/index.js [hunk 2]
```diff
@@ -71,3 +71,20 @@ export function readOnly(target, name, descriptor) {
...
+export var observes = decoratorAlias(Ember.observer, 'Connot `observe` without property names');
```
Comment (rwjblue) [inline addon/index.js]: s/Connot/Can not/

**Comment by rwjblue** [comment]
Looks like we have an issue with the 1.10 tests, though not directly with this PR.  I'll open a separate issue for that.

**Comment by rwjblue** [comment]
@stefanpenner - a few cleanup bits here, but otherwise good to go

**Comment by stefanpenner** [comment]
updated based on comments

**Comment by rwjblue** [comment]
Thank you!

**Pull request merged**